client/src/main/java/org/evosuite/testcase/ExecutableChromosome.java

```diff
@@ -34,7 +34,8 @@ public abstract class ExecutableChromosome<E extends ExecutableChromosome<E>> ex
 
     private static final long serialVersionUID = 1L;
 
-    protected transient ExecutionResult lastExecutionResult = null;
+    //protected transient ExecutionResult lastExecutionResult = null;
+    protected ExecutionResult lastExecutionResult = null;
```

Comment:
Why does not the `lastExecutionResult` need to be `transient`?

Reply:
The same explanation as "ExecutionResult.java"